src/popup/popup.js

```
@@ -53,15 +140,119 @@
+            } else if (ruleTypeParam === '2') {
+                bmRuleType = 'constraint';
+                logDebug("Detected configuration constraint rule");
+            } else if (ruleTypeParam === '11') {
+                bmRuleType = 'hiding';
+                logDebug("Detected configuration hiding rule");
+            }
+            else if (ruleTypeParam === '23') {
+                bmRuleType = 'bommapping';
+                logDebug("Detected configuration bommapping rule");
+            } else {
+                bmRuleType = 'other_rule_type_' + ruleTypeParam;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)